clash-lib/src/Clash/Normalize/Transformations.hs

```diff
@@ -497,11 +497,11 @@ caseCon ctx@(TransformContext is0 _) e@(Case subj ty alts)
                 let e' = mkApps (Prim nm ty') [repTy,Right ty,msgOrCallStack]
                 in  changed e'
             (Prim nm ty',[_])
-              | nm `elem` ["Clash.Transformations.undefined"] ->
+              | nm `elem` ["Clash.Transformations.undefined", "Clash.GHC.Evaluator.undefined"] ->
                 let e' = mkApps (Prim nm ty') [Right ty]
                 in changed e'
             (Prim nm _,[_])
-              | nm `elem` ["EmptyCase"] ->
+              | nm == "EmptyCase" ->
```

Comment:
This entire case-alternative is actually a latent bug; the `EmptyCase` primitive should be in the same list as `Clash.Transformation.undefined`. See also https://github.com/clash-lang/clash-compiler/commit/32489b7a7b91f80a7fdcb4dcb3b88b366cd2dfae

Reply:
Alright, but should that go in this PR? (Also: that change was accidentally committed after playing with hlint / intellij integration.)

Reply:
If it's an accidental commit, remove it. Otherwise, fix it properly :)

Reply:
Fair enough!